src/js/components/Tabs/Tabs.js

```diff
@@ -114,7 +83,21 @@ const Tabs = forwardRef(
           gap={theme.tabs.gap}
           {...tabsHeaderStyles}
         >
-          {tabs}
```

Comment:
For readability, I kind of liked having `tabs` defined above.

Consider `const tabs = React.Children.map( ...`

Reply:
Good clean up idea.